lib/dor/text_extraction/ocr.rb

```diff
@@ -38,15 +38,13 @@ def cleanup
           tries += 1
           sleep(3**tries)
 
-          raise e unless tries < 4
+          retry if tries < max_tries
 
-          Honeybadger.notify('[NOTE] Problem deleting files and folders in ocrWF:ocr-workspace-cleanup', context: { druid: bare_druid, tries:, error: e })
-          retry
+          raise e
```

Comment:
do not send HB alerts if the output folder retried, we will still get an HB alert if it actually fails after multiple tries